src/components/HeroStrategy/hero-right.tsx

```diff
@@ -38,85 +40,119 @@ export const HeroStrategyRight: VFC<HeroStrategyRightProps> = ({
-          />
-        </VStack>
+      {countdown && (
+        <Box
+          background="surface.tertiary"
+          padding="30px 16px"
+          borderRadius={34}
+          borderWidth={1}
+          borderColor="surface.tertiary"
+          fontWeight="bold"
+          textAlign="center"
+          fontSize="3xl"
+        >
+          Coming soon
+        </Box>
+      )}
+      {!countdown && (
+        <BaseButton
+          w="full"
+          h="50px"
+          onClick={() => {
+            analytics.track("strategy.buy-sell", {
+              strategyCard: cellarData.name,
+              landingType: landingType(),
+            })
+            buyOrSellModal.onOpen()
+          }}
+        >
+          Buy / Sell
+        </BaseButton>
+      )}
+      {!countdown && (
+        <BuyOrSellModal
+          uniswapLink={content.buyUrl}
+          id={id}
+          isOpen={buyOrSellModal.isOpen}
+          onClose={buyOrSellModal.onClose}
+        />
+      )}
+      {!countdown && (
+        <Link
+          href={`/strategies/${id}/manage`}
+          onClick={() => {
+            analytics.track("strategy.manage-portfolio", {
+              strategyCard: cellarData.name,
+              landingType: landingType(),
+            })
+          }}
+          style={{ textDecoration: "none" }}
+        >
+          <SecondaryButton w="full" h="50px">
+            Manage Portfolio
+          </SecondaryButton>
+        </Link>
+      )}
+      {!countdown && (
+        <HStack
+          pt={4}
+          justifyContent="space-around"
+          alignItems="start"
+          divider={<StackDivider borderColor="purple.dark" />}
+        >
+          <VStack flex={1}>
+            <Heading size="md">{tokenPrice || "--"}</Heading>
+            <CellarStatsLabel
+              tooltip="The dollar value of the ETH, BTC, and USDC that 1 token can be redeemed for"
+              title="Token Price"
+            />
+          </VStack>
 
-        <VStack flex={1}>
-          <PercentageText data={dailyChange} headingSize="md" />
-          <CellarStatsLabel
-            tooltip="% change of current token price vs. token price yesterday"
-            title="1D Change"
-          />
-        </VStack>
+          <VStack flex={1}>
+            <PercentageText data={dailyChange} headingSize="md" />
+            <CellarStatsLabel
+              tooltip="% change of current token price vs. token price yesterday"
+              title="1D Change"
+            />
+          </VStack>
 
-        <VStack flex={1} textAlign="center">
-          {intervalGainPct.data ? (
-            <PercentageText
-              data={intervalGainPct.data}
-              headingSize="md"
+          <VStack flex={1} textAlign="center">
+            {intervalGainPct.data ? (
+              <PercentageText
+                data={intervalGainPct.data}
+                headingSize="md"
+              />
+            ) : (
+              <Box>--</Box>
+            )}
```

Comment:
`PercentageText` component already handle null values so we don't need to make a ternerary options
```suggestion
            <PercentageText
                data={intervalGainPct.data}
                headingSize="md"
              />
```